generated/elasticsearch/README.md

```diff
@@ -8,35 +8,162 @@ please check out [USAGE.md](../../USAGE.md).
 
 ## Notes on index naming
 
-This sample Elasticsearch template will apply to any index named `try-ecs-*`.
+These sample Elasticsearch templates will apply to any index named `try-ecs-*`.
 This is good for experimentation.
 
 Note that an index following ECS can be named however you need. There's no requirement
 to have "ecs" in the index name.
 
 ## Instructions
 
+Elasticsearch 7.8 introduced
+[composable index templates](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-templates.html)
+as the new default way to craft index templates.
+
+The following instructions let you use either approach.
+
+### Composable index templates
+
+**Warning**: The artifacts based on coposable templates are newly introduced in the ECS repository.
+Please try them out and give us feedback if you encounter any issues.
+
+If you want to play with a specific version of ECS, check out the proper branch first.
+Note that the composable index templates are available in the ECS 1.7 branch or newer.
+
+```
+git checkout 1.7
```

Comment:
I didn't test extensively, but `--ref` worked as well in a basic test.

Reply:
The goal of these instructions is for anyone to be able to come here and grab the reference templates without needing to install or build anything.

I see the usage.md docs and the CLI as a more powerful tool that power users can bend to their will, but these are optional.